payload-schemas/schemas/installation_repositories/removed.schema.json

```diff
@@ -120,6 +120,22 @@
     },
     "repository_selection": { "type": "string" },
     "repositories_added": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["id", "node_id", "name", "full_name", "private"],
+        "properties": {
+          "id": { "type": "integer" },
+          "node_id": { "type": "string" },
+          "name": { "type": "string" },
+          "full_name": { "type": "string" },
+          "private": { "type": "boolean" }
+        },
+        "additionalProperties": false
+      },
+      "maxItems": 0
+    },
```

Comment:
In theory here, this would the it's proper type, however as this is the removed action schema, this array should be empty, which is why `maxItems` is set to 0